code/src/index.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
```

Comment:
Should be a space between import ` import ReactDOM from 'react-dom'` and import './index.css' 🥳